src/components/ScrollToTop.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component} from 'react';
+import {Component} from 'react';
```

Comment:
Can you double-check that this compiles/runs? I think it should work since there's no jsx below but I want to be extra-sure

Reply:
It compiles/runs fine. Had made changes to remove 
![image](https://user-images.githubusercontent.com/42149840/62814241-791fe300-bac4-11e9-8c8e-b9e6f7544b5e.png)